newa/__init__.py

```diff
@@ -730,6 +747,18 @@ def id(self) -> str:
         raise NotImplementedError
 
 
+class NVRParser:
```

Comment:
Maybe this need not to be a class? Do we need it for something else than to get name from a build? If no then I suggest to have a method for that only.

Reply:
In fact, I was thinking there could be some methods in the future (maybe some NVR arithmetics etc.), just don't have a use case for it right now. 
Right now we need it for name but at the same time this class provides also version and release.
I think I like the object approach a bit more.
@happz Opinions?

Reply:
Done.